_layouts/startup.html

```diff
@@ -57,6 +57,53 @@ <h2>À propos</h2>
             <a class="fr-link fr-link--icon-left fr-icon-mail-line" href="mailto:{{ site.email }}?subject={{ page.title | uri_escape }}%20sur%20beta.gouv.fr">Contacter l'incubateur</a></em>
             {% endif %}
           </p>
+
+          <div class="fr-my-2w startup-evaluation">
+            <h6>Standards de qualité</h6>
```

Comment:
```suggestion
            <h3 class="fr-h6">Standards de qualité</h6>
```
Pour respecter la hiérarchie des titres

Reply:
Côté hiérarchie de l'information, y a un espace trop important en ce titre et son contenu ; et ça rend moins lisible ce qui est associé à quoi. La classe "fr-my-1w" pourrait aider mais c'est agaçant de devoir faire ça.